rome-modules/src/main/java/com/rometools/modules/atom/modules/AtomLinkModule.java

```diff
@@ -13,16 +13,19 @@
  * limitations under the License.
  *
  */
+
 package com.rometools.modules.atom.modules;
 
 import com.rometools.rome.feed.atom.Link;
 import com.rometools.rome.feed.module.Module;
 
+import java.util.List;
+
 public interface AtomLinkModule extends Module {
 
-    public static final String URI = "http://www.w3.org/2005/Atom";
+    String URI = "http://www.w3.org/2005/Atom";
```

Comment:
Maybe he wanted to save 3 words? 😉 Nevermind , variables in interfaces are always public static final